### DISCUSSION TITLE
How to make code area wider or code size smaller?

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#2606] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/66788741/87819648-9283f000-c83a-11ea-8e41-4dc8b8854595.png)
I don't like how long functions are cut off.

**Comment by mmistakes** [comment, marked as answer]
The code block width fills the space of it's parent container... the main content.
If you want to increase that use the [wide single layout variant](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#wide-page), it will give you some more space.

Otherwise your only other solution is to modify the theme's CSS to wrap codes which would avoid the need for the horizontal scrollbar.

This CSS should do the trick:

```css
pre {
  white-space: pre-wrap;
}
```
